iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/ReactiveCapabilityCurveImpl.java

```diff
@@ -135,4 +169,22 @@ public double getMaxQ(double p) {
             }
         }
     }
+
+    @Override
+    public double getMinQ(double p, boolean extrapolateReactiveLimitSlope) {
+        checkPointsSize(points);
+        if (!extrapolateReactiveLimitSlope || p >= points.firstKey() && p <= points.lastKey()) {
+            return getMinQ(p);
+        }
+        return extrapolateReactiveLimitsSlope(p).getMinQ();
+    }
+
+    @Override
+    public double getMaxQ(double p, boolean extrapolateReactiveLimitSlope) {
+        checkPointsSize(points);
+        if (!extrapolateReactiveLimitSlope || p >= points.firstKey() && p <= points.lastKey()) {
```

Comment:
getMaxP() and getMinP() would be easier to read and maintain data encapsulation.